src/hearings/converters/hearing-panel-required.converter.ts

```diff
@@ -0,0 +1,25 @@
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { State } from '../store';
+import { AnswerConverter } from './answer.converter';
+
+export class HearingPanelRequiredConverter implements AnswerConverter {
+  constructor(protected readonly route: ActivatedRoute) {}
+
+  public transformAnswer(hearingState$: Observable<State>): Observable<string> {
+    return hearingState$.pipe(
+      map((state) => {
+        if (
+          !state.hearingRequest.hearingRequestMainModel.hearingDetails ||
+          !state.hearingRequest.hearingRequestMainModel.hearingDetails
+            .panelRequirements
+        )
+          return;
+        return state.hearingRequest.hearingRequestMainModel.hearingDetails.panelRequirements.panelPreferences.length > 0
```

Comment:
How about if there are some judges users inside this panelRequirements.panelPreferences, and then this is showing yes, but the memberType here is for PANEL_MEMBER only, so it needs to filter that.